app/domain/export/pdf/passes/membership/person.rb

```diff
@@ -66,9 +66,10 @@ def membership_verify_token
       person.membership_verify_token.presence || person.init_membership_verify_token!
     end
 
-    def group_name
+    def layer_name
       group = person.primary_group.presence || person.groups.first
-      group.short_name.presence || group.name
+      layer = group.layer_group
+      layer.short_name.presence || layer.name
```

Comment:
Das ist ziemlich genau die Implementierung von `display_name`, daher hätte ich da einen Vorschlag :grin: 

```suggestion
      group.layer_group.display_name
```